tests/packagedcode/test_package_models.py

```diff
@@ -22,23 +22,146 @@
 #  ScanCode is a free software code scanning tool from nexB Inc. and others.
 #  Visit https://github.com/nexB/scancode-toolkit/ for support and download.
 
-from __future__ import absolute_import, print_function
+from __future__ import absolute_import
+from __future__ import print_function
 
 import os.path
 
 from commoncode.testcase import FileBasedTesting
 
+from packagedcode.models import AssertedLicense
 from packagedcode import models
-from collections import OrderedDict
+from packagedcode.models import Package
+from packagedcode.models import Party
 
 
 class TestModels(FileBasedTesting):
     test_data_dir = os.path.join(os.path.dirname(__file__), 'data')
 
     def test_model_creation_and_dump(self):
-        aap = models.AndroidAppPackage()
+        aap = models.AndroidAppPackage(name='someAndroidPAcakge')
         result = aap.as_dict()
-        expected = OrderedDict([('type', 'Android app'),
-                                ('packaging', 'archive'),
-                                ('primary_language', 'Java')])
+        expected = {
```

Comment:
Actually, this is unlikely yield a stable ordering:
https://github.com/nexB/scancode-toolkit/blob/829ea55b4d11e4c8ad379a2e081cd7f590b07e81/src/packagedcode/models.py#L306.
If `self.to_primitive().items()` return a bare dict, then you are not really creating an ordereddict with a stable order. You can wrapping a random dict in an ordereddict. Still random in the end.